Date: Tue, Sep 20 2011 10:52:10
From: Seth Heasley <seth.heasley@intel.com>
References: bnc#682755 fate#311837
Subject: New RAID DeviceID for Intel Panther Point PCH
Patch-mainline: 2cab7a4c    3.1-rc1

Add a new RAID DeviceID for the Intel Panther Point PCH.

This patch adds a new DeviceID for SATA RAID on the Intel Panther Point PCH.
The DeviceID was recently added to documentation but has always been in the
silicon.

Signed-off-by: Seth Heasley <seth.heasley@intel.com>
Acked-by: Jiri Kosina <jkosina@suse.com>

---

--- linux-3.0.4-0.11/drivers/ata/ahci.c.orig	2011-10-30 20:57:25.000000000 -0700
+++ linux-3.0.4-0.11/drivers/ata/ahci.c	2011-10-30 20:57:36.000000000 -0700
@@ -267,6 +267,7 @@
 	{ PCI_VDEVICE(INTEL, 0x1e05), board_ahci }, /* Panther Point RAID */
 	{ PCI_VDEVICE(INTEL, 0x1e06), board_ahci }, /* Panther Point RAID */
 	{ PCI_VDEVICE(INTEL, 0x1e07), board_ahci }, /* Panther Point RAID */
+	{ PCI_VDEVICE(INTEL, 0x1e0e), board_ahci }, /* Panther Point RAID */
 
 	/* JMicron 360/1/3/5/6, match class to avoid IDE function */
 	{ PCI_VENDOR_ID_JMICRON, PCI_ANY_ID, PCI_ANY_ID, PCI_ANY_ID,
